docs/rulesets.md

```diff
@@ -1,41 +1,128 @@
-spectral lint foo.yaml --ruleset=path/to/acme-company-ruleset.yaml --ruleset=http://example.com/acme-common-ruleset.yaml
+To customize the rules that are applied, simply create a `spectral.yml` in the same directory that you are running the `spectral lint` command from and it will automatically be used.
+
+## Ruleset Examples
+
+Spectral currently support ruleset files in both `yaml` and `json` formats.
+
+### Adding a rule
+
+Add your own rules under the `rules` property in your `spectral.yml` ruleset file.
+
+**spectral.yml**
+
+```yaml
+rules:
+  my-rule-name:
+    description: Tags must have a description.
+    given: $.tags[*]
+    severity: error
+    then:
+      field: description
+      function: truthy
+```
+
+The example above adds a single rule that checks that tags objects have a description property defined.
+
+Running `spectral lint` on the following object with the rulset above will result in an error being reported, since the tag does not have a description:
+
+```json
+{
+  "tags": [{
+    "name": "animals"
+  }]
+}
+```
+
+While running it with this object will succeed:
+
+```json
+{
+  "tags": [{
+    "name": "animals",
+    "description": "come in all shapes and sizes"
+  }]
+}
+```
+
+### Adding to the recommended OpenAPI rules
+
+Spectral comes with two built in rulesets - one for OpenAPI v2 (`spectral:oas2`), and one for OpenAPI v3 (`spectral:oas3`). Use the `extends` property in your ruleset file to build upon or customize other rulesets.
+
+**spectral.yml**
+
+```yaml
+extends: spectral:oas2
+rules:
+  my-rule-name:
+    description: Tags must have a description.
+    given: $.tags[*]
+    then:
+      field: description
+      function: truthy
+```
+
+The example above will apply the recommended rules from the built in OpenAPI v2 ruleset AND apply the custom `my-rule-name` rule.
+
+### Enabling specific OpenAPI rules
+
+Sometimes you might want to apply specific rules from another ruleset. Use the `extends` property, and pass `off` as the second argument in order to load add the rules from another ruleset, but disable them all by default. This allows you to pick and choose which rules you would like to enable.
+
+**spectral.yml**
+
+```yaml
+extends: [[spectral:oas2, off]]
+rules:
+  # This rule is defined in the spectral:oas2 ruleset. We're passing `true` to turn it on and inherit the severity defined in the spectral:oas2 ruleset.
+  operation-operationId-unique: true
+```
+
+The example above will run the two rules we enabled, since we pass `off` to disable all rules by default when extending the `spectral:oas2` ruleset.
+
+### Disabling specific OpenAPI rules
+
+This example shows the opposite of the `Enabling specific OpenAPI rules` example. Sometimes you might want to enable all rules by default, and disable a few.
+
+**spectral.yml**
+
+```yaml
+extends: [[spectral:oas2, all]]
+
+# we could also just extend spectral:oas2 like this, which would only run the recommended rules (rather than every single rule)
```

Comment:
I'm confused on how I will know the recommended rules.

Reply:
A great question - we should probably add another article or two that render the openapi v2 and v3 rulesets in a table or something. One of the table columns could indicate if the rule is "recommended" or not.

Reply:
Even if it just means pointing to a file for now where they are, it is better than them being kind of a mystery to the user.

Reply:
Done!